test/unit/MessageComposer/attachmentManager.test.ts

```diff
@@ -575,6 +576,35 @@ describe('AttachmentManager', () => {
   });
 
   describe('getUploadConfigCheck', () => {
+    it('should not check for file extension config if its a Blob type', async () => {
+      const blob = new Blob([''], { type: 'image/gif' });
+      const {
+        messageComposer: { attachmentManager },
+      } = setup();
+      const result = await attachmentManager.getUploadConfigCheck(blob);
+      expect(result).not.toEqual({
+        uploadBlocked: false,
+        reason: 'allowed_file_extensions',
+      });
+    });
+
+    it('should check for file extension config if its a FileReference type', async () => {
```

Comment:
We should be probably checking the the same results we get for File and FileReference.